README.md

```diff
@@ -41,8 +41,8 @@ It offers a variety of features:
 * [RocketMQ Client Go](https://github.com/apache/rocketmq-client-go)
 * [RocketMQ Client Python](https://github.com/apache/rocketmq-client-python)
 * [RocketMQ Client Nodejs](https://github.com/apache/rocketmq-client-nodejs)
-* [RocketMQ Spring](https://github.com/apache/rocketmq-spring)
-* [RocketMQ Exporter](https://github.com/apache/rocketmq-exporter)
+* [RocketMQ Spring](https://github.com/apache/rocketmq-spring): The integration component of RocketMQ and spring framework
```

Comment:
1. 'spring' should be 'Spring'. 
2. Accurately rocketmq-spring is starter of Spring Boot, not integration to Spring Framework itself.